flink-scala-shell/src/main/scala/org/apache/flink/api/scala/FlinkILoop.scala

```diff
@@ -139,7 +147,8 @@ class FlinkILoop(
     "org.apache.flink.api.scala._",
     "org.apache.flink.api.scala.utils._",
     "org.apache.flink.streaming.api.scala._",
-    "org.apache.flink.streaming.api.windowing.time._"
+    "org.apache.flink.table.api._",
+    "org.apache.flink.table.api.scala._"
```

Comment:
Could we also import `org.apache.flink.types.Row` which is used very frequently I believe  ?

Reply:
Agree +1，I will add it.